nb-configuration.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project-shared-configuration>
```

Comment:
You need to remove this file from PR commit. If you want you can open a new pull request!